docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -409,10 +410,28 @@ The following table lists the measurement data that the HRM LED green sensor pro
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
 | values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
 
+<a name="hrm_green_batch"></a>
+## Heart rate monitor LED green batch Sensor
```

Comment:
@mijin85cho I wish to get your suggestion on the following: 
- As I suggested the developer to use the sentence case in headings, what should be done to the description as in the following case: 

1
The Heart Rate Monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor buffers and sends the recorded data in batches. The sensor must be power-efficient and support batch processing.

or 

2
The heart rate monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor buffers and sends the recorded data in batches. The sensor must be power-efficient and support batch processing.



I would prefer 1st one. 
What is your suggestion?

Reply:
@nawab-samsung There are no big difference. Both are acceptable.

Reply:
@mijin85cho thanks. 
@bbrto21 then we can keep the current version.

Reply:
@nawab-samsung ok, is there anything to update more?